ppl/archive/import.go

```diff
@@ -153,7 +153,7 @@ func (dw *tsDirWriter) addBufSize(delta int64) {
 // totalRecordBytes is the sum of the size of compressed records spilt to disk
 // and a crude approximation of the buffer record bytes (simply bufBytes / 2).
 func (dw *tsDirWriter) totalRecordBytes() int64 {
-	b := dw.bufSize
+	b := dw.bufSize / 2
```

Comment:
dw.spiller.SpillSize reports uncompressed bytes, so should we do `return b /2 ` instead of this?

Reply:
Oh, snap I'm realizing that `spill` doesn't compress the spills. Ok.